### ISSUE TITLE
Copying Config Between Machines

### ISSUE THREAD

**Issue by harisekhon** [erikdoe/ccmenu#47]
I've been digging around to try to copy my CCMenu config between machines and found the following file:
```
~/Library/Containers/net.sourceforge.cruisecontrol.CCMenu/Data/Library/Preferences/net.sourceforge.cruisecontrol.CCMenu.plist
```
Copying this to another machine however hasn't worked, although it's the only file I found while grepping that contained my travis URLs... and removing the file does wipe out the configuration so I'm sure it's being read.

Is this a general non-portability issue with the plist format or am I missing something?

It would be nice if configuration management of CCMenu was documented.

**Comment by erikdoe** [comment]
You've got the right file and copying it should work. I know it worked before CCMenu became sandboxed.

For this particular issue: can you confirm that CCMenu was not running on the target machine when you copied the file?

For the general issues of multi-machine / user synchronisation, can we keep this in #24?

**Comment by harisekhon** [comment]
After a large amount of testing this evening, git, ```md5``` to prove the files were the same, even handing editing the binary file using plist editor from XCode I determined that there was no way CCMenu was respecting that plist file as removing a repo in the binary editor and resaving didn't make the repo disappear from CCMenu after start . I even wiped out the cache db files in case they were caching it. I then even wiped out the whole Library/Containers/net.sourceforge.cruisecontrol.CCMenu and reseeded the plist file and it still didn't work.

It turns out my hunch around it being configured or cached elsewhere was correct - after more digging it turns out Mavericks onwards has a system wide cache that indeterminately gets refreshed. I did a killall of cfprefsd which forced a reload of the cache and now all my repos appear in CCMenu, so I'm happy I have a portable way of migrating this config.
